### ISSUE TITLE
Continuing to index files in directories that are excluded after being excluded

### ISSUE THREAD

**Issue by addshore** [DEVSENSE/phptools-docs#106]
I'm trying out a local history plugin today https://marketplace.visualstudio.com/items?itemName=xyz.local-history

I installed the plugin, made some code edits, and then realized that phptools was indexing the .history directory, which lead to some fun suggestions and ui docs.

See below phpdoc suggestions for previous versions of this method

![image](https://user-images.githubusercontent.com/3308769/103633300-73d6fa00-4f3d-11eb-8029-48a9e67e3f27.png)

I added the .history directory to the 2 exclusion lists that I was expecting would work, but the files still appear to be indexed by phptools

![image](https://user-images.githubusercontent.com/3308769/103633403-95d07c80-4f3d-11eb-9f4f-4ad30f025132.png)

But the files continued to appear

![image](https://user-images.githubusercontent.com/3308769/103633607-e34ce980-4f3d-11eb-9a6b-eff80fcbd590.png)

**Comment by jakubmisek** [comment]
`files.exclude` should be supported, we'll take a look on that!

Maybe, for now, it may help to close a reopen the folder in vscode?

**Comment by jakubmisek** [comment]
Ok, seems VSCode sends us file-change-events even the file is excluded. Working on it.

**Comment by addshore** [comment]
> Ok, seems VSCode sends us file-change-events even the file is excluded. Working on it.

Once restarting the editor the previously indexed files (that I had actually deleted) were removed from the phptools index.
I guess ad the index is built from scratch then.

But it would be good when changing the setting to actively then remove the directories from the index.
I guess the file-change-events issue you identified is similar but perhaps slightly different?
